spec/system/author_visits_the_home_page_spec.rb

```diff
@@ -73,4 +73,26 @@
       end
     end
   end
+
+  context "author isn't the one who posted the comment" do
+    it "shouldn't show the edit link" do
+      user = create(:user)
+      other_user = create(:user)
+      article = create(
+        :article,
+        content: "My awesome content",
+        title: "This is my awesome title",
+        user: user
+      )
+
+      visit article_path(article, as: user)
+      fill_in "comment_body", with: "This is a sample comment"
+      click_button "Submit"
+
+      visit article_path(article, as: other_user)
```

Comment:
Could you replace all this with the following?

```ruby
comment = create :comment

visit article_path comment.article, as: other_user
```

Reply:
This has been addressed in #225